src/CMakeLists.txt

```diff
@@ -7,16 +7,28 @@ elseif(UNIX)
 endif()
 
 if(CMAKE_C_COMPILER_ID MATCHES "^(Intel)$")
-  set(CMAKE_Fortran_FLAGS "-g -fpp -free -xHOST -traceback -FR")
-  set(fortran_8_flags "-i8 -r8")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
```

Comment:
Don't think this is necessary.

Reply:
Right, I see that now. If nothing overwrites the flag it's fine.